go.mod

```diff
@@ -5,20 +5,24 @@ go 1.12
 require (
 	github.com/btcsuite/btcd v0.0.0-20190315201642-aa6e0f35703c // indirect
 	github.com/btcsuite/btcutil v0.0.0-20190316010144-3ac1210f4b38 // indirect
-	github.com/cosmos/cosmos-sdk v0.34.0
+	github.com/cosmos/cosmos-sdk v0.0.0-00010101000000-000000000000
```

Comment:
Will these reflect changes for 34.7? Or has the local branch been merged with the recent changes

Reply:
In below, we move to support v0.34.7 and custom msg codec. 

`replace github.com/cosmos/cosmos-sdk => github.com/YunSuk-Yeo/cosmos-sdk v0.34.7-terra`

If you mind to use personal github repository, we can move to terra-project repository.